volumes/vds/sql/functions/create-function-partition_vdsdata.sql

```diff
@@ -0,0 +1,104 @@
+/*
+Parameters:
+Name | Type | Description
+base_table | text | The parent table under which the new partition(s) would be created, e.g., `vds.raw_vdsdata_div8001`
+year_ | integer | The year whose data would be saved in the new partition, e.g., 2023
+div_id | integer | Division_id, used to determine one or two yearly partitions, e.g., 8001
+
+Return: Void
+Purpose:
+Create new partitions for the given division_id and year (if needed) under the parent table `base_table`
+*/
```

Comment:
Thanks for adding this Sphinx-friendly block comment. You can find more about these comments at https://pypi.org/project/sphinx-sql/